staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/resource_access_test.go

```diff
@@ -121,5 +125,112 @@ func TestLRUCacheLookup(t *testing.T) {
 			}
 		})
 	}
+}
+
+func TestGetQuotas(t *testing.T) {
+	namespace := "test"
+	namespace1 := "test1"
+	resourceQuota := &corev1.ResourceQuota{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+	}
+
+	resourceQuotas := []*corev1.ResourceQuota{resourceQuota}
+
+	liveLookupCache := lru.New(10000)
```

Comment:
why do we need a larger cache than the default?

Reply:
Good question - this was also copy/pasta.
I ran this test on master (without the changes to make it pass) using `lru.New(100)` (the default) and it failed too, so I'll use 100 here as well for consistency. Thanks for pointing this out.